Scheduling.c

```diff
@@ -97,7 +97,7 @@ Panel* Scheduling_newPriorityPanel(int policy, int preSelectedPriority) {
    return this;
 }
 
-bool Scheduling_setPolicy(Process* proc, Arg arg) {
+bool Scheduling_setPolicy(Row* row, Arg arg) {
```

Comment:
Keep API to use processes. Scheduling NICs or namespaces does not make sense. It's the task for the caller to ensure we have a process.

Reply:
Done, using same model as Affinity.c discussion earlier now.